### DISCUSSION TITLE
New release please

### DISCUSSION THREAD

**Discussion by sagarishere** [recharts/recharts#5349] [General]
Nextjs 15.1 has stably started supporting React 19. 

I can see that the dependency have been updated here also, however, the new version with the dependency update has not been released. Kindly release the minor version update asap.

**Comment by ckifer** [comment]
https://github.com/recharts/recharts/issues/4558#issuecomment-2536714016

Please be patient with me as this project is maintained in spare time only

**Reply by sagarishere** [reply]
Thank you for such fast speed. You are amazing
